src/search/pdbs/pdb_heuristic.cc

```diff
@@ -33,13 +33,27 @@ int PDBHeuristic::compute_heuristic(const State &ancestor_state) {
     return h;
 }
 
+static basic_string<char> paper_references() {
+    return utils::format_conference_reference(
+        {"Stefan Edelkamp"},
+        "Planning with Pattern Databases",
+        "https://ecp01.icaps-conference.org/papers/A-longpapers/2/edelkamp.pdf",
+        "Proceedings of the Sixth European Conference on Planning (ECP 2001)",
+        "84--90",
+        "AAAI Press",
+        "2001");
+}
 class PDBHeuristicFeature
     : public plugins::TypedFeature<Evaluator, PDBHeuristic> {
 public:
     PDBHeuristicFeature() : TypedFeature("pdb") {
         document_subcategory("heuristics_pdb");
         document_title("Pattern database heuristic");
-        document_synopsis("TODO");
+        document_synopsis(
+            "Computes dictionaries for state-to-goal distances in"
+            "state space abstractions based on projections."
+            "First used in domain idependent planning by:"
+            + paper_references());
```

Comment:
There are spaces missing, so I think this will generated "instate" and "projections.First".
Typo "idependent", and I think "domain-independent" should be written with a hyphen, not as two words.

I don't think people describe PDBs as "dictionaries" (in particular because they don't store keys, only values). I think it's not necessary to get into such details in the first place, this could just be changed to "Computes goal distance in state space abstractions based on projections" etc.).

The particular PDB implementation in Fast Downward has its own paper (by Ortlieb et al.), which I think is worth referencing. It is already referenced from another PDB-related plug-in, see https://www.fast-downward.org/Doc/PatternCollectionGenerator#Hill_climbing

Reply:
I mean the paragraph "For implementation notes, see:" followed by the Ortlieb et al. reference. Most of that paper is actually about the basic PDB implementation, not the hill-climbing pattern collection paper. (But the paper discusses that too, so it's not wrong that the paper is also referenced from that plugin.)